pkg/apiserver/apiserver.go

```diff
@@ -213,7 +213,8 @@ func newAPIHandlerEngine() (apiHandlerEngine *gin.Engine, endpoint *gin.RouterGr
 	apiHandlerEngine.Use(apiutils.MWHandleErrors())
 
 	endpoint = apiHandlerEngine.Group("/dashboard/api")
-	endpoint.StaticFS("/speedscope", speedscopeFiles.Assets())
+	ssEndpoint := apiHandlerEngine.Group("/dashboard/speedscope")
+	ssEndpoint.StaticFS("/", speedscopeFiles.Assets())
```

Comment:
I think you should either move these two lines out of `apiserver`, or rename this function, as it is not an API, but clearly static assets.